src/test/scala/com/microsoft/partnercatalyst/fortis/spark/SparkSpec.scala

```diff
@@ -1,18 +1,27 @@
-package com.microsoft.partnercatalyst.fortis.spark.streamprovider
+package com.microsoft.partnercatalyst.fortis.spark
 
-import org.apache.spark.{SparkConf, SparkContext}
+import java.time.Instant
+import java.util.UUID
+import java.util.UUID.randomUUID
+
+import com.datastax.spark.connector._
+import com.microsoft.partnercatalyst.fortis.spark.dto.{Analysis, AnalyzedItem}
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.{CassandraConfig, CassandraSink}
+import com.microsoft.partnercatalyst.fortis.spark.streamprovider.{ConnectorConfig, StreamFactory, StreamProvider, UnsupportedConnectorConfigException}
 import org.apache.spark.rdd.RDD
-import org.apache.spark.streaming.{Seconds, StreamingContext}
 import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import org.apache.spark.{SparkConf, SparkContext}
 import org.scalatest.{BeforeAndAfter, FlatSpec}
 
 import scala.collection.mutable
 
-class StreamProviderSpec extends FlatSpec with BeforeAndAfter {
+class SparkSpec extends FlatSpec with BeforeAndAfter {
```

Comment:
I would ideally like to keep the StreamProviderSpec separate since the StreamProvider package is an isolated component that could be moved out to a library at any time (and it'd be nice to take the test spec with it).